.github/workflows/pythontest.yml

```diff
@@ -0,0 +1,84 @@
+name: Python tests
+
+on: [push, pull_request]
+
+jobs:
+  pre_job:
+    name: Path match check
+    runs-on: ubuntu-latest
+    # Map a step output to a job output
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@master
+        with:
+          github_token: ${{ github.token }}
+          paths: '["**.py", "requirements.txt", "requirements-dev.txt", ".github/workflows/pythontest.yml"]'
```

Comment:
Looks like we removed the docker-compose baed tests, is that right?

Reply:
Yes, it was simpler to not encapsulate the tests inside docker, and allows us to take better advantage of dependency caching.